realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -820,6 +820,7 @@ public static String getTableName() {
         RealmList<some.test.AllTypes> columnRealmListList = realmObjectSource.realmGet$columnRealmList();
         if (columnRealmListList != null) {
             RealmList<some.test.AllTypes> columnRealmListRealmList = realmObjectCopy.realmGet$columnRealmList();
+            columnRealmListRealmList.clear();
```

Comment:
The list will be created before when create `Object4957.child`. https://github.com/realm/realm-java/pull/5023/files/caac7175c9074e42aa23d657be8e193dbf576d04#diff-693596b26c39426696c02bf1cbf4a647R28 It is quite a minor case.